src/customers/account-dashboard-order-by-sku.md

```diff
@@ -3,6 +3,13 @@ ee_only: true
 title: Order by SKU
 ---
 
+SKU stands for Stock Keeping Unit. SKUs help online sellers identify the most important product characteristics such as: size, color, price, material etc. Don't confuse product IDs with SKUs:
+
+    - The `Product ID` is a set of numbers that are used internally to identify products and are not available to customers.
```

Comment:
It's worth mentioning that a Product ID in Magento is fixed, can never be changed and is set by the system. 
```suggestion
    - The `Product ID` is sequential series of numbers that are used internally to identify products and are not available to customers.
```